about/team/index.md

```diff
@@ -2,61 +2,109 @@
+  project and responsible for making new releases (or guilty of not making
+  them in time, depending on the point of view). Vadim is based in France and
+  runs his own consultancy company, [TT-Solutions](https://www.tt-solutions.com/).
+
+* [Dr. Stefan Csomor](mailto:csomor@advancedconcepts.ch) - An actual medical
+  doctor, Stefan Csomor wrote and maintains wxMac, the port to OS X, and wxiOS.
+  He runs [Advanced Concepts AG](http://www.advanced.ch/), a Swiss software
   consultancy specialising in medical software.
 
+* [Paul Cornett](mailto:wxml-paul@bullseye.com) - Paul is the maintainer of
+  wxGTK.
+
+* [Robin Dunn](mailto:robin@alldunn.com) - Robin works on the very popular
+  [wxPython](http://wxpython.org/) project, which allows using wxWidgets from
+  Python, and also contributed many fixes and improvements to wxWidgets
+  itself.
+
+* [Václav Slavík](mailto:vaclav.slavik@matfyz.cz) - Václav worked on wxHTML, the
+  XRC system, the bakefile-based build system and many other areas. He is also
+  the author of [Poedit](https://poedit.net/), the powerful and intuitive
+  translation editor.
+
+* [Bryan Petty](mailto:bryan@ibaku.net) - Bryan contributed to several areas
+  of wxWidgets and has taken care of hosting wxTrac (now retired), wxWiki and
+  wxWidgets documentation site, without speaking of this website itself, for
+  many years.
+
+
+## Other Active Developers
+
+Thanks to the following people for their continued contributions to wxWidgets:
+
+* Artur Wieczorek who is the maintainer of wxPropGrid library and the resident
+  graphics expert.
+
+* Maarten Bent who is a very regular contributor to wxMSW, implementing big
+  parts of high DPI support in it, maintainer of the CMake-based build system
+  and much more.
+
+* Tobias Taschner who has added several entire components of wxWidgets,
+  including, but not limited to, `wxWebRequest` and the related classes, and
+  `wxWebViewEdge` backend.
+
+* Scott Talbert who added EGL support in `wxGLCanvas` and more.
+
+* Ulrich Telle who added `wxUILocale` class and contributed other fixes.
+  Ulrich is also the author of several popular wxWidgets-related libraries,
+  such as [wxSQLite3](https://github.com/utelle/wxsqlite3),
+  [wxPdfDocument](https://github.com/utelle/wxpdfdoc) and
+  [wxChartDir](https://github.com/utelle/wxchartdir).
+
+* PB has contributed many miscellaneous improvements to wxWidgets and
+  its documentation but also numerous valuable contributions to GitHub reviews
+  and issues discussions.
+
+* Jouk Jansen maintains wxWidgets for OpenVMS systems.
+
+* Danny Scott and Xavier Perrissoud build wxMSW binaries for wxWidgets
+  releases.
+
+* Special thanks to Eric Jensen for his always pertinent contributions to the
+  various mailing list discussions.
+
+* Many more people ought to be listed here but for now just a few more
+  names and thanks to: AliKet, Andreas Falkenhahn, Daniel Kulp, Ian McInerney,
```

Comment:
`Ali Kettab` is my full name
aliket.83@gmail.com is my email
thank you for including my name in the list by the way

Reply:
Thanks, I knew your name but as your public profile didn't mention it, I didn't know whether you wanted to use it or not -- I'll use it now that you've agreed to it.

I've decided not to add emails for everybody (and probably should remove the existing emails as I suspect that half of them isn't valid any more...), but please let me know and I can add a link with your email if you'd like to have it here.

Reply:
Just the name is enough then, thank you.